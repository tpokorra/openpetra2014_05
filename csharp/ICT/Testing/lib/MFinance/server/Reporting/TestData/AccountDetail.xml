<?xml version="1.0"?>
<Parameters>
  <Parameter id="xmlfiles" value="eString:Finance\\accountdetail.xml,Finance\\accountdetailcommon.xml,Finance\\finance.xml,common.xml" />
  <Parameter id="currentReport" value="eString:Account Detail" />

  <Parameter id="param_diff_period_i" value="eInteger:0" />
  <Parameter id="param_account_hierarchy_c" value="eString:STANDARD" />
  <Parameter id="param_currency" value="eString:Base" />
  <Parameter id="param_period" value="eString:true" />
  <Parameter id="param_date_checked" value="eBoolean:false" />
  <Parameter id="param_start_period_i" value="eInteger:1" />
  <Parameter id="param_end_period_i" value="eInteger:1" />

  <Parameter id="param_year_i" value="eInteger:0" />
  <Parameter id="param_sortby" value="eString:Account" />
  <Parameter id="param_account_codes" value="eString:*NOTUSED*" />
  <Parameter id="param_account_code_start" value="eString:&quot;0100&quot;" />
  <Parameter id="param_account_code_end" value="eInteger:9800" />
  <Parameter id="param_rgrAccounts" value="eString:AccountRange" />
  <Parameter id="param_cost_centre_codes" value="eString:*NOTUSED*" />
  <Parameter id="param_cost_centre_code_start" value="eString:&quot;0400&quot;" />
  <Parameter id="param_cost_centre_code_end" value="eInteger:9500" />
  <Parameter id="param_rgrCostCentres" value="eString:CostCentreRange" />
  <Parameter id="param_depth" value="eString:standard" />


  <Parameter id="param_quarter" value="eString:false" />
  <Parameter id="param_daterange" value="eString:false" />
  <Parameter id="MaxDisplayColumns" value="eInteger:7" />

  <Parameter id="ColumnPosition" column="0" value="eInteger:0" />
  <Parameter id="ColumnPositionIndented" column="0" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="0" value="eInteger:1" />
  <Parameter id="param_calculation" column="0" value="eString:Transaction Cost Centre" />

  <Parameter id="ColumnPosition" column="1" value="eInteger:1" />
  <Parameter id="ColumnPositionIndented" column="1" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="1" value="eInteger:1" />
  <Parameter id="param_calculation" column="1" value="eString:Transaction Account Code" />

  <Parameter id="ColumnPosition" column="2" value="eInteger:6" />
  <Parameter id="ColumnPositionIndented" column="2" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="2" value="eInteger:2" />
  <Parameter id="param_calculation" column="2" value="eString:Debit" />

  <Parameter id="ColumnPosition" column="3" value="eInteger:8" />
  <Parameter id="ColumnPositionIndented" column="3" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="3" value="eInteger:2" />
  <Parameter id="param_calculation" column="3" value="eString:Credit" />

  <Parameter id="ColumnPosition" column="4" value="eInteger:10" />
  <Parameter id="ColumnPositionIndented" column="4" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="4" value="eInteger:5" />
  <Parameter id="param_calculation" column="4" value="eString:Transaction Narrative" />

  <Parameter id="ColumnPosition" column="5" value="eInteger:15" />
  <Parameter id="ColumnPositionIndented" column="5" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="5" value="eInteger:2" />
  <Parameter id="param_calculation" column="5" value="eString:Start Balance" />

  <Parameter id="ColumnPosition" column="6" value="eInteger:17" />
  <Parameter id="ColumnPositionIndented" column="6" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="6" value="eInteger:2" />
  <Parameter id="param_calculation" column="6" value="eString:End Balance" />

  <Parameter id="param_with_analysis_attributes" value="eString:false" />
</Parameters>