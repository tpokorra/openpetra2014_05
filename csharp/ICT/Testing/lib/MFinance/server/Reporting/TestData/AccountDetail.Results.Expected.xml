id,Details,Left2,CC,Acc,Debits,Credits,Narrative,Start Balance,End Balance
{ledgernumber}00/0100/Credit/1/1/4,Credit 31/01/{ThisYear},,{ledgernumber}00,"0100",,-100.00,Gift Example,,
{ledgernumber}00/0100/Detail,Sub Total:,,,,0.00,-100.00,,0.00,100.00
{ledgernumber}00/0100/Net Balance,Net Balance:,,,,,-100.00,,,
"0100",Account Total:,,,,0.00,-100.00,,0.00,100.00
0100/Net Balance,Net Balance:,,,,,-100.00,,,
{ledgernumber}00/4200/Debit/1/1/6,Debit 31/01/{ThisYear},,{ledgernumber}00,4200,20.00,,Expense Example,,
{ledgernumber}00/4200/Detail,Sub Total:,,,,20.00,0.00,,0.00,-20.00
{ledgernumber}00/4200/Net Balance,Net Balance:,,,,20.00,,,,
4200,Account Total:,,,,20.00,0.00,,0.00,-20.00
4200/Net Balance,Net Balance:,,,,20.00,,,,
{ledgernumber}00/6000/Debit/1/1/3,Debit 31/01/{ThisYear},,{ledgernumber}00,6000,100.00,,Gift Example,,
{ledgernumber}00/6000/Credit/1/1/5,Credit 31/01/{ThisYear},,{ledgernumber}00,6000,,-20.00,Expense Example,,
{ledgernumber}00/6000/Detail,Sub Total:,,,,100.00,-20.00,,0.00,-80.00
{ledgernumber}00/6000/Net Balance,Net Balance:,,,,80.00,,,,
6000,Account Total:,,,,100.00,-20.00,,0.00,-80.00
6000/Net Balance,Net Balance:,,,,80.00,,,,
{ledgernumber}00/6200/Debit/1/1/1,Debit 31/01/{ThisYear},,{ledgernumber}00,6200,40.00,,Start Balance,,
{ledgernumber}00/6200/Detail,Sub Total:,,,,40.00,0.00,,0.00,-40.00
{ledgernumber}00/6200/Net Balance,Net Balance:,,,,40.00,,,,
6200,Account Total:,,,,40.00,0.00,,0.00,-40.00
6200/Net Balance,Net Balance:,,,,40.00,,,,
{ledgernumber}00/9700/Credit/1/1/2,Credit 31/01/{ThisYear},,{ledgernumber}00,9700,,-40.00,Start Balance,,
{ledgernumber}00/9700/Detail,Sub Total:,,,,0.00,-40.00,,0.00,40.00
{ledgernumber}00/9700/Net Balance,Net Balance:,,,,,-40.00,,,
9700,Account Total:,,,,0.00,-40.00,,0.00,40.00
9700/Net Balance,Net Balance:,,,,,-40.00,,,
GrandTotal,Grand Total:,,,,160.00,-160.00,,0.00,0.00
GrandTotalNetBalance,Net Balance:,,,,,0.00,,,
